spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../solver'
+
+describe Solver do
+  describe '#fizzbuzz' do
+    context 'When providing an integer number, check if it is divisible by both 3 and 5' do
+      # Arrange
+      let(:solver) { Solver.new }
+
+      it 'returns "fizz" when divisible by 3' do
+        # Act
+        divisible3 = 3
+
+        # Assert
+        expect(solver.fizzbuzz(divisible3)).to eq('fizz')
+      end
+
+      it 'returns "buzz" when divisible by 5' do
+        # Act
+        divisible5 = 5
+
+        # Assert
+        expect(solver.fizzbuzz(divisible5)).to eq('buzz')
+      end
+
+      it 'returns "fizzbuzz" when divisible by both 3 and 5' do
+        # Act
+        divisible3_and5 = 15
+
+        # Assert
+        expect(solver.fizzbuzz(divisible3_and5)).to eq('fizzbuzz')
+      end
```

Comment:
You didn't implement a test case for the last condition in the fizzbuzz test cases 👍 

![image](https://github.com/Ruthmy/TDD-Practice/assets/20537712/8de3982f-253f-4640-9bd3-765cb837e2f5)

Reply:
@usorfaitheloho Thanks for your feedback. We'll check it!